src/dataLoaders/actions/dataLoaders.ts

```diff
@@ -407,12 +408,21 @@ export const createOrUpdateTelegrafConfigAsync = () => async (
   )
 
   if (telegrafConfigID) {
-    const telegraf = await client.telegrafConfigs.update(telegrafConfigID, {
-      name: telegrafConfigName,
-      description: telegrafConfigDescription,
-      plugins,
+    const response = await putTelegraf({
+      telegrafID: telegrafConfigID,
+      data: {
+        name: telegrafConfigName,
+        description: telegrafConfigDescription,
+        // TODO: interface TelegrafPlugin does not have property 'type'
+        plugins,
```

Comment:
We may want to update the type of `plugins` which is interface `TelegrafPlugin`.